Add a new public key

deovaliandro opened the pull request:
pub   ed25519/B905A377C78E929C 2021-07-24 [SC]
      90263F5E34B5D26F841B2D70B905A377C78E929C
uid                 [  full  ] Deo Valiandro. M (Wibu) <deo@gnuweeb.org>
uid                 [  full  ] Deo <deo@gnuweeb.org>
sub   cv25519/7F997A7D409F53B7 2021-07-24 [E]

Signed-off-by: deovaliandro <deo@gnuweeb.org>

Commit:
Add a new public key

pub   ed25519/B905A377C78E929C 2021-07-24 [SC]
      90263F5E34B5D26F841B2D70B905A377C78E929C
uid                 [  full  ] Deo Valiandro. M (Wibu) <deo@gnuweeb.org>
uid                 [  full  ] Deo <deo@gnuweeb.org>
sub   cv25519/7F997A7D409F53B7 2021-07-24 [E]

Signed-off-by: deovaliandro <deo@gnuweeb.org>

ammarfaizi2 commented on keys/90263F5E34B5D26F841B2D70B905A377C78E929C.asc:
```diff
@@ -0,0 +1,11 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+xjMEYPuc/RYJKwYBBAHaRw8BAQdA2f29j7PbuAHzJ/qce4wHGxOa/kY+JhYqk972GJZkvPPNFURl
```
Please add the key info here as well.

deovaliandro replied:
fix: https://github.com/GNUWeeb/pgpkeys/pull/6/commits/2a41258be7f71d5fe07d7d919c81a54ffa22d3db

ammarfaizi2 reviewed (changes requested):
Please add the key info in the beginning of the line.

Commit:
Add a new public key

pub   ed25519/B905A377C78E929C 2021-07-24 [SC]
      90263F5E34B5D26F841B2D70B905A377C78E929C
uid                 [  full  ] Deo Valiandro. M (Wibu) <deo@gnuweeb.org>
uid                 [  full  ] Deo <deo@gnuweeb.org>
sub   cv25519/7F997A7D409F53B7 2021-07-24 [E]

Signed-off-by: deovaliandro <deo@gnuweeb.org>

ammarfaizi2 reviewed (approved):
Good, I am merging this.
